app/page-objects/page.js

```diff
@@ -0,0 +1,7 @@
+class Page {
+    open(url) {
+      return browser.url(url);
+    }
+  }
```

Comment:
Please format your code. And please check all files in all pull requests.
Some useful link: https://www.digitalocean.com/community/tutorials/how-to-format-code-with-prettier-in-visual-studio-code